airflow/providers/google/cloud/sensors/bigquery.py

```diff
@@ -147,8 +147,11 @@ def __init__(
         gcp_conn_id: str = "google_cloud_default",
         delegate_to: str | None = None,
         impersonation_chain: str | Sequence[str] | None = None,
+        deferrable: bool = False,
         **kwargs,
     ) -> None:
+        if deferrable and "poke_interval" not in kwargs:
+            kwargs["poke_interval"] = 5
```

Comment:
Don’t need to be done in this PR but should be a good idea to define `5` as a constant somewhere (maybe `BaseSensorOperator.DEFERRABLE_POLL_INTERVAL`) so we don’t repeat the magic number everywhere.

Reply:
I just noticed [AzureDataFactoryPipelineRunStatusAsyncSensor](https://github.com/apache/airflow/blob/3239720c6e9f8a868c6a74871be24049b367aa34/airflow/providers/microsoft/azure/sensors/data_factory.py#L102) using 60 instead of 5. Do you think we still should add this constant in this case?